### DISCUSSION TITLE
Temporarily hiding the Snap Panel

### DISCUSSION THREAD

**Discussion by bb222nm** [rxhanson/RectanglePro-Community#187] [Q&A]
Is it possible to temporarily hide the Snap Panel while dragging a window around? (like by holding a modifier key, perhaps?)
Having it appear next to the cursor is very handy but sometimes it gets in the way. On the same note, could the style of the panel (i.e., size of the elements) get change?

![side-panel](https://user-images.githubusercontent.com/98620610/207903481-c4cfc5a2-3658-431b-b45c-ccdcfbf6e812.jpg)

**Comment by rxhanson** [comment]
Excellent point. Would you rather have it only appear while holding a modifier, rather than not appear while holding a modifier?

It's on my backlog to allow adjusting of the size of the panel elements.

**Comment by bb222nm** [comment]
I think having it not appear while holding a modifier would be better since it's handy most often.
Keep up the great work!

**Comment by solid-pixel** [comment]
I second this request, it's really the only gripe (and a big one) I have with Rectangle Pro :)

I'm fine with the option suggested, though I really miss what PowerToys allows to do, which is using Right Click as a modifier.

The way it works is:
- You start dragging a window by holding down the Left Click as usual
- With Left Click down, you hit Right Click
- The Snap Panel pops up

Either way thank you @rxhanson for this fantastic piece of software, it's just fantastic ❤️

**Reply by rxhanson** [reply]
Thanks for the feedback! I'll give this a shot when I work on this feature. Looks like it'll be a 2-3 releases away at this point.

**Reply by solid-pixel** [reply]
Thanks man!

**Comment by CrescentMadeJr** [comment]
I was coming here to put in for this.  There are some good suggestions.  I would prefer no modifier.  I would like if the panel just showed a little bit when dragging around the window and when I hover over that then it pops out all the way for me to chose the layout.  See here just after the 3.02 mark in the video for what I mean.  https://youtu.be/wtZRpThnbTM?t=182

Thanks for all your hard work.

**Comment by solid-pixel** [comment]
That is a great idea, but wouldn’t it get in the way of the area at the top
you can hit to maximise the window? Or with the macOS thingy (forgot the
name) to move a window to another desktop?

**Reply by CrescentMadeJr** [reply]
You're right, it could.  Maybe having it drop just a bit more.  Or I might be just happy with a modifier.

**Reply by rxhanson** [reply]
I could still make something that gets close to what's in Windows but doesn't get in the way. I'll have to play around with it and see. Thanks for the feedback!
